netci.groovy

```diff
@@ -1064,11 +1064,15 @@ combinedScenarios.each { scenario ->
 
                                     // Up the timeout for arm64 jobs.
                                     Utilities.setJobTimeout(newJob, 240);
-                                    buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${architecture} /toolset_dir C:\\ats"
 
-                                    // Debug runs take too long to run.
-                                    if (lowerConfiguration != "debug") {
-                                       buildCommands += "Z:\\arm64\\common\\scripts\\arm64PostBuild.cmd %WORKSPACE% ${architecture} ${lowerConfiguration}"
+                                    // Debug runs take too long to run. So build job only.
+                                    if (lowerConfiguration == "debug") {
+                                       buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${architecture} /toolset_dir C:\\ats"
+                                    }
+                                    else {
+                                       buildCommands += "set __TestIntermediateDir=int&&build.cmd skiptests ${lowerConfiguration} ${architecture} /toolset_dir C:\\ats"
+                                       // Test build and run are launched together.
```

Comment:
Yes it's disabled in build.cmd but it's enabled in the subsequent script (arm64PostLauncher.cmd) which I added in CI script, which will launch buildtest.cmd